untrusted/quote_verify/Cargo.toml

```diff
@@ -4,8 +4,11 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-pem = "1.0.2"
 sha2 = "0.10.2"
-mbedtls = "0.8.1"
 hex = "0.4.3"
 displaydoc = "0.2.3"
+p256 = { version = "0.11.1", features = [ "ecdsa", "pem" ] }
+x509-parser = { version = "0.14.0", features = [ "verify" ] }
```

Comment:
This is fine for now, but long-term we won't be able to use the "verify" feature on that crate, because that uses `ring` as a dependency.